color-coordinated/color-picker-fe
# Sk component testing

#### What's this PR do?

Add component, mapState, and mapDispatch testing for App, ProjectCard, ProjectContainer, all reducers, and SaveProjectForm.

#### Where should the reviewer start?

I would start with the reducers, as those are the most straight forward, and then move through the component tests.

#### How should this be manually tested?

Check out to the branch, run `npm test` and ensure all the tests pass for you as well.

#### Any background context you want to provide?

Lots of appreciation for helping me to remember to pass all the props!

#### Screenshots (if appropriate)
<img width="707" alt="Screen Shot 2019-12-11 at 11 19 20 AM" src="https://user-images.githubusercontent.com/46407593/70648415-3e5fc000-1c08-11ea-8cc8-ec53775174c1.png">

#### Questions:
There's a test file for index.test.js that's failing. I don't believe we even need this file. Thoughts?
